### PR TITLE
Fix get active networks early returns

### PR DESCRIPTION
**Pull request by hyphenized**
Small fix, added a test too!

## To Test
- [ ] Add the pair of brackets back, test should break

### CODE REVIEW

File: background/services/chain/tests/index.unit.test.ts [hunk 1]
```diff
@@ -79,4 +80,48 @@ describe("Chain Service", () => {
...
     })
   })
+
+  describe("getActiveNetworks", () => {
+    let chainService: ChainService
+
+    beforeEach(async () => {
+      sandbox.restore()
+      chainService = await createChainService()
+    })
```
Comment (0xDaedalus) [inline background/services/chain/tests/index.unit.test.ts:90]: Hmmmn we should really only need this once in this file.

I think what we want to do here is move lines 20-26 into the top-level "Chain Service" describe block.

**Review by 0xDaedalus — COMMENTED** [review]
💯 📈 Fantastic Catch.  

If you wouldn't mind quickly re-arranging the `beforeEach` I'm eager to get this guy merged.

**Comment by hyphenized** [comment]
@0xDaedalus Updated the `markNetworkActivity` test to use sinon fake timers because it was flaky on my end (Maybe because of the Date stub 😖)

**Review by greg-nagy — APPROVED** [review]
this is a neat fix, let's 🚢

**Pull request merged**